libs/sdk-ui-dashboard/src/model/events/index.ts

```diff
@@ -244,20 +241,6 @@ export {
     DashboardDrillToLegacyDashboardRequested,
     DashboardDrillToLegacyDashboardResolved,
     DashboardDrillableItemsChanged,
-    drillRequested,
-    drillResolved,
-    drillDownRequested,
-    drillToAttributeUrlRequested,
-    drillToCustomUrlRequested,
-    drillToDashboardRequested,
-    drillToInsightRequested,
-    drillToLegacyDashboardRequested,
-    drillDownResolved,
-    drillToAttributeUrlResolved,
-    drillToCustomUrlResolved,
-    drillToDashboardResolved,
-    drillToInsightResolved,
-    drillToLegacyDashboardResolved,
```

Comment:
just for confirmation.. are you removing these in favor of command started events which Dan started last week or so?

Reply:
These should not be public.